README.md

```diff
@@ -104,6 +104,39 @@ cd build.mingw
 cmake .. -DCMAKE_TOOLCHAIN_FILE=../../../mingww.toolchain -DSDL2_DIR=/usr/local/cross-tools/x86_64-w64-mingw32/lib/cmake/SDL2
 ```
 
+### Building & Running on OSX
+
+You will need build tools and CMAKE. Assuming you have [homebrew](https://docs.brew.sh/Installation) installed
+
+``` shell
+xcode-select --install
+brew install clang
```

Comment:
There isn't a `clang` formula in homebrew-core, clang is installed by the `llvm` formula - but the xcode-select will do so as well - I assume you've used that rather than brew here?

Reply:
Doh - that line was supposed to be `cmake`. Yup  I used xcode-select for the tools